modules/core/shared/src/main/scala/lucuma/core/model/Extinction.scala

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats.kernel.Order
+import eu.timepit.refined.types.numeric.NonNegShort
+import io.circe.Decoder
+import io.circe.Encoder
+import lucuma.core.optics.Format
+import monocle.Prism
+
+/** 
+ * Extinction in mags, a non-negative number with two decimal points of precision, 
+ * in [0.00, 327.67].
+ */
+opaque type Extinction = NonNegShort
+
+object Extinction:
+
+  def apply(millimags: NonNegShort): Extinction =
+    millimags
+
+  val FromMillimags: Prism[Short, Extinction] =
+    Prism((s: Short) => NonNegShort.from(s).toOption)(_.value)
+
+  val FromMags: Format[Double, Extinction] =
```

Comment:
Should it be `Format[BigDecimal, Extinction]`?

Reply:
Well it totally works. I must have been hallucinating.